docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs

```diff
@@ -66,25 +50,25 @@ public static void Example()
             // ----------
         }
 
-        /// <summary>
-        /// inputSize is the overall dimensions of the model input tensor.
-        /// </summary>
+        // <summary>
```

Comment:
i mean let's delete the xml *tags* like <summary> and just keep the comment text:

// inputSize is the overall dimensions of the model input tensor.